### ISSUE TITLE
Error: EACCES: permission denied, scandir

### ISSUE THREAD

**Issue by alexgorbatchev** [fwouts/previewjs#1535]
### Describe the bug

I have a few docker volume mounts in the workspace, preview.js fails to start because it can't traverse into those folders:

```
500 in endpoint /previews/start:
[Error: EACCES: permission denied, scandir '...'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'scandir',
  path: '...
}
```

### Reproduction

1. chown a folder to another user
2. attempt to review

### Preview.js CLI version

not sure where it is, just installed it via VSCode

### Framework

react 18

### System Info

```shell
n/a
```

### Used Package Manager

pnpm

**Comment by alexgorbatchev** [comment]
Another thing I want to point out, it's trying to scan folders OUTSIDE of the workspace it found:

i have a `web` folder which is a root folder for the node.js project in my work space, I see:

```
✨ Created Preview.js workspace for: /home/b/web
```
`EACCES` error is for `/home/a/folder`.

**Comment by alexgorbatchev** [comment]
and one more thing, the folder it's trying to scan is in `.gitignore`

**Comment by fwouts** [comment]
Thanks @alexgorbatchev, will investigate!

**Comment by fwouts** [comment]
This appears to be a bug in https://github.com/sindresorhus/globby (see https://github.com/fwouts/globby-eaccess-bug for repro), but the good news is I think I know how to fix it. Let's hope they accept my PR once it's ready :)

**Comment by alexgorbatchev** [comment]
@fwouts excellent find! 👏

**Comment by fwouts** [comment]
Hey @alexgorbatchev, this should now be fixed in v1.21. Let me know if that works for you!
